tests/test_update.py

```diff
@@ -52,6 +54,44 @@ def test_update_with_fork_point_not_specified(self, mocker: Any) -> None:
              "'git rebase' to the parent branch of the current branch."
              )
 
+    def test_update_drops_empty_commits(self, mocker: Any) -> None:
+        """Verify behaviour of a 'git machete update' command.
+
+        Verify that 'git machete update' drops effectively-empty commits.
+        """
+        git = GitContext()
+        if git.get_git_version() < (2, 26, 0):
```

Comment:
Actually... maybe even I'd handle the case of git < 2.26.0 separately in the test. `git machete update` should fail, but then after `git rebase --continue`, the desired state should still be achieved 🤔